spec/amp-var-substitutions.md

```diff
@@ -65,6 +65,15 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?path=%2Fpage1.html`.
 
+**CLIENT_ID**
+
+Use the special string `CLIENT_ID` to add a per AMP document source origin client identifier.
```

Comment:
You set yourself up for quite the hornet nest :)
The first arg isn't an analytics provider. Internally we call it cid-scope. We also need to document that this is literally falls back to a cookie of that very name when not served from the cache.